client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx

```diff
@@ -48,8 +48,10 @@ const BusinessPlanDetails = ( { selectedSite, sitePlans, selectedFeature } ) =>
 
 			<PurchaseDetail
 				icon="stats-alt"
-				title={ i18n.translate( 'Stats from Google Analytics' ) }
-				description={ i18n.translate( 'Connect to Google Analytics for the perfect complement to WordPress.com stats.' ) }
+				title={ i18n.translate( 'Connect to Google Analytics' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Connect Google Analytics' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/checkout-thank-you-business-analytics-feature-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).